build-release-tools/docker_build.sh

```diff
@@ -64,6 +64,9 @@ doBuild() {
             exit 1
         fi
         pushd $repo
+            # Add version/commit in commitstring.txt for docker image build.
+            git log -n 1 --pretty=format:%h.%ai.%s > commitstring.txt
```

Comment:
where will the commitstring.txt finally be located ?
Will you also update the release confluence page to reflect this? I meant this one : https://rackhd.atlassian.net/wiki/spaces/RAC1/pages/20519777/RackHD+Release+Installation+Guide#RackHDReleaseInstallationGuide-sprintrelease.4